File: repasoHerenciajava/Alumno.java
import java.util.ArrayList;
class Alumno extends Personal {
    String semestre;
    ArrayList<String> materiasCursando;
    public Alumno(String name, String email, String password, String semestre) {
        super(name, email, password);
        this.semestre = semestre;
        this.materiasCursando = new ArrayList<String>();
    }
}

File: repasoHerenciajs/alumno.js
import Personal from './personal.js'
class Alumno extends Personal {
    constructor(name, email, password, age, semestre) {
        super(name, email, password, age);
        this.materiasCursando = new Array();
        this.semestre = semestre;
    }
}

export default Alumno;